src/utils.js

```diff
@@ -2,6 +2,7 @@ import _ from 'lodash/fp'
 import F from 'futil'
 import { useState } from 'react'
 import { mapProps } from 'recompose'
+import { loremIpsum } from 'lorem-ipsum'
```

Comment:
this is in src/utils, but `lorem-ipsum` should not be bundled with grey-vest

Reply:
moved to stories/utils